### ISSUE TITLE
Changes to GPRs for Cytochrome C Oxidoreductase (AKA Complex IV of the Electron Transport Chain)

### ISSUE THREAD

**Issue by Devlin-Moyer** [SysBioChalmers/Human-GEM#526]
The GPRs for MAR06914 and MAR13081 (cytochrome c oxidoreductase; complex IV of the electron transport chain) were recently changed from

> ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and ENSG00000131143 and ENSG00000131055 and ENSG00000178741 and ENSG00000135940 and ENSG00000111775 and ENSG00000156885 and ENSG00000126267 and ENSG00000160471 and ENSG00000164919 and ENSG00000161281 and ENSG00000112695 and ENSG00000131174 and ENSG00000170516 and ENSG00000127184 and ENSG00000176340 and ENSG00000187581 and ENSG00000189043

to

> ENSG00000111775 and ENSG00000112695 and ENSG00000126267 and ENSG00000127184 and ENSG00000131143 and ENSG00000131174 and ENSG00000135940 and ENSG00000164919 and ENSG00000176340 and ENSG00000178741 and ENSG00000198712 and ENSG00000198804 and ENSG00000198938 and ENSG00000189043

but several of the genes that were removed can functionally replace one of the genes that remain, and are often expressed in a mutually-exclusive fashion across different tissues. [source](https://doi.org/10.1155/2017/1534056) Specifically:

- COX4I1 (ENSG00000131143) is ubiquitously expressed and COX4I2 (ENSG00000131055) is only expressed in hypoxic conditions 
- COX6A1 (ENSG00000111775) is generally expressed everywhere but muscle tissues and COX6A2 (ENSG00000156885) is generally only expressed in muscle tissues
- COX6B1 (ENSG00000126267) is expressed in somatic tissues and COX6B2 (ENSG00000160471) is only expressed in testes
- COX7A1 (ENSG00000161281) is generally only expressed in muscle tissues and COX7A2 (ENSG00000112695) is generally expressed everywhere but muscle tissues
- COX8A (ENSG00000176340) is ubiquitously expressed and COX8C (ENSG00000187581) is only expressed in some tissues, although its functional role is apparently unknown, so maybe it should stay out of this GPR

~Also, [NDUFA4 (ENSG00000189043) is required for wild-type catalytic activity of Complex IV](https://doi.org/10.1016/j.tem.2018.03.009), but is currently absent from the GPRs of both reactions~

So the GPRs for MAR06914 and MAR13081 should be changed to:

> ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and (ENSG00000131143 or ENSG00000131055) and ENSG00000178741 and ENSG00000135940 and (ENSG00000111775 or ENSG00000156885) and (ENSG00000126267 or ENSG00000160471) and ENSG00000164919 and (ENSG00000161281 or ENSG00000112695) and ENSG00000131174 and ENSG00000170516 and ENSG00000127184 and (ENSG00000176340 or ENSG00000187581) and ENSG00000189043

or

> ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and (ENSG00000131143 or ENSG00000131055) and ENSG00000178741 and ENSG00000135940 and (ENSG00000111775 or ENSG00000156885) and (ENSG00000126267 or ENSG00000160471) and ENSG00000164919 and (ENSG00000161281 or ENSG00000112695) and ENSG00000131174 and ENSG00000170516 and ENSG00000127184 and ENSG00000176340 and ENSG00000189043

depending on what you think about COX8C (ENSG00000187581) having an "unknown functional role"

**Comment by haowang-bioinfo** [comment]
After #489 , the current GPR of MAR06914 has been changed as below: 
> "ENSG00000111775 and ENSG00000112695 and ENSG00000126267 and ENSG00000127184 and ENSG00000131143 and ENSG00000131174 and ENSG00000135940 and ENSG00000164919 and ENSG00000176340 and ENSG00000178741 and ENSG00000198712 and ENSG00000198804 and ENSG00000198938 and ENSG00000189043"

please double check, maybe update the requets

**Comment by Devlin-Moyer** [comment]
Ah sorry for missing that; I briefly checked the GPR for MAR06914 in the latest release of Human1 and when I saw it was still all ANDs and no ORs I assumed it was entirely the same

I see that you added NDUFA4 and dropped COX4I2, COX6A2, COX6B2, COX7A1, COX7B2, and COX8C, citing the subunit composition of [this entry](https://www.ebi.ac.uk/complexportal/complex/CPX-6123) on the complex portal.
One of the two citations on that page is [this paper](http://europepmc.org/abstract/MED/30030361), which ends the introduction to its section on Complex IV by saying "Complex IV is the only OXPHOS complex containing tissue-specific and developmentally regulated isoforms [118-119], reflecting the importance of an exquisite regulation of COX activity." and reference 118 is [the first paper I cited](https://doi.org/10.1155/2017/1534056) in the initial message here.

The reason why that complex portal entry dropped all of those subunits is because each individual Complex IV will have _either_ COX4I1 (in normoxic conditions) _or_ COX4I2 (in hypoxic conditions), and either COX6A1 (in non-muscle tissues) or COX6A2 (in muscle tissues), etc.; that page describes one particular Complex IV assembly that only includes the most common/"canonical" member of each pair.

I think it is important to "or" together each of these pairs as I initially proposed, since many of those pairs are usually expressed in a mutually-exclusive fashion, and with the current GPR, if you tried to make a muscle-specific GEM from Human-GEM using RNA-seq/proteomics data from actual muscle samples, you might run into a problem with this reaction, cuz the muscles probably wouldn't express any/much COX6A1 or COX7A2, but there'll still likely be plenty of fully-functional Complex IV that just has COX6A2 and COX7A1 in their place (see section 5.2 [here](https://doi.org/10.1155/2017/1534056). If you think it's necessary, I can try to see if there are any papers where someone isolated and/or crystallized human Complex IV with one of the subunit substitutions I mentioned, but I think the paper I cited initially demonstrates that there is functional Complex IV in tissues that don't express one of the genes in the current GPR but do express one of the genes you removed because it can perform the same role.

**Comment by haowang-bioinfo** [comment]
> If you think it's necessary, I can try to see if there are any papers where someone isolated and/or crystallized human Complex IV with one of the subunit substitutions I mentioned

It makes sense to include tissue/stage-specific subunits in the GPR. Please go ahead to collect these alternative subunits @Devlin-Moyer

**Comment by Devlin-Moyer** [comment]
By "collect these alternative ones" do you mean "propose a new GPR that includes them appropriately"? If so:

ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and (ENSG00000131143 or ENSG00000131055) and ENSG00000178741 and ENSG00000135940 and (ENSG00000111775 or ENSG00000156885) and (ENSG00000126267 or ENSG00000160471) and ENSG00000164919 and (ENSG00000161281 or ENSG00000112695) and ENSG00000131174 and ENSG00000170516 and ENSG00000127184 and ENSG00000176340 and ENSG00000189043

**Comment by haowang-bioinfo** [comment]
> do you mean "propose a new GPR that includes them appropriately"? or so:
>
> ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and (ENSG00000131143 or ENSG00000131055) and ENSG00000178741 and ENSG00000135940 and (ENSG00000111775 or ENSG00000156885) and (ENSG00000126267 or ENSG00000160471) and ENSG00000164919 and (ENSG00000161281 or ENSG00000112695) and ENSG00000131174 and ENSG00000170516 and ENSG00000127184 and ENSG00000176340 and ENSG00000189043


if this is your final suggestion?

**Comment by Devlin-Moyer** [comment]
it is; it's got every pair I originally mentioned "or"ed together except COX8A and COX8C, since, upon closer investigation, I determined that the function of COX8C in humans is currently unknown

**Comment by haowang-bioinfo** [comment]
why are there two Complex IV reactions ([MAR06914](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR06914) and [MAR13081](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR13081)) included, anyone has a clue about this?

**Comment by Devlin-Moyer** [comment]
MAR06914 is the "normal" complex IV reaction, and MAR13081 is almost identical except instead of consuming 8 H<sup>+</sup> and producing 2 H<sub>2</sub>O, it consumes 7.92 H<sup>+</sup> and produces 1.96 H<sub>2</sub>O and 0.02 O<sub>2</sub><sup>-</sup>, presumably as some sort of attempt to model reactive oxygen species production by the electron transport chain. I have no idea where those stoichiometric coefficients came from, and MAR13081 has no references associated with it

**Comment by haowang-bioinfo** [comment]
> I have no idea where those stoichiometric coefficients came from, and MAR13081 has no references associated with it

neither am I - how about just remove it (Occam's razor principle)

**Comment by haowang-bioinfo** [comment]
@feiranl @JonathanRob @avlant how about take away MAR13081?

**Comment by feiranl** [comment]
@johan-gson has checked the ROS, maybe know the reason?

**Comment by haowang-bioinfo** [comment]
> Yes the additional complex IV reaction is to represent the low level of ROS byproduct generation during its normal function.

@JonathanRob good to know this, also both can be mapped to counterparts in Recon3D:
[MAR06914](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR06914) -> [CYOOm2i](https://metabolicatlas.org/identifier/Recon3D/CYOOm2i)
[MAR13081](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR13081)) -> [CYOOm3i](https://metabolicatlas.org/identifier/Recon3D/CYOOm3i)

**Comment by johan-gson** [comment]
@feiranl This is the ROS AddOn I have been talking about before - i.e., it would be nice if the model was without ROS in it's normal use, but that ROS could be added to reactions if desired. This is a lot of work though. I personally always remove this reaction in the model preparation step, so I recommend removing it for now. Many of the oxphos complexes generates ROS, so why only keep this one? I think complex I is the worst, and that this is one possible explanation for why complex I is bypassed at high growth rates, in addition to the high enzyme usage per produced ATP?

**Comment by haowang-bioinfo** [comment]
> it would be nice if the model was without ROS in it's normal use, but that ROS could be added to reactions if desired.

@johan-gson nice input - on what conditions the ROS should be added back?

**Comment by johan-gson** [comment]
I'm just thinking if someone wants to model ROS, it would be nice to have some kind of addon so it could be added to the model. But I don't have a concrete suggestion on exactly how.

**Comment by haowang-bioinfo** [comment]
> Many of the oxphos complexes generates ROS, so why only keep this one? I think complex I is the worst, and that this is one possible explanation for why complex I is bypassed at high growth rates, in addition to the high enzyme usage per produced ATP?

@johan-gson have you attempted to add complex I version reaction for ROS production, and how are the stoichiometric coefficients determined?

**Comment by feiranl** [comment]
If we want to provide a ROS version, it is better to remove that before use rather supply them as addons. I would vote for keep them here, but remove them in the tINIT part for specific model generation.

**Comment by johan-gson** [comment]
Yeah, maybe you are right, it would make sense to have the complete model and a way to remove them. Hao, I have not tried to add the ROS, I have been more interested in removing those reactions so far. But they are probably important for the modeling if constrained in a good way.

**Comment by haowang-bioinfo** [comment]
> I have not tried to add the ROS, I have been more interested in removing those reactions so far.

thanks Johan

> I would vote for keep them here

Keeping ROS version might be reasonable because it: i) is useful information to a knowledgebase; ii) enables potential simulation of oxidative stress with the existing [ROS detoxification](https://metabolicatlas.org/explore/Human-GEM/gem-browser/subsystem/ros_detoxification) subsystem in HumanGEM.

> but remove them in the tINIT part for specific model generation

My only suggestion is to block it (lb=ub=0) for now, this would avoid it carrying fluxes in normal use and tINIT.

**Comment by haowang-bioinfo** [comment]
fixed in #532
